src/PaymentGateways/Gateways/Stripe/resources/js/StripeAccountPanel.tsx

```diff
@@ -0,0 +1,108 @@
+import {__} from '@wordpress/i18n';
+import {InspectorControls} from '@wordpress/block-editor';
+import {Notice, PanelBody, SelectControl, ToggleControl} from '@wordpress/components';
+import {createInterpolateElement} from '@wordpress/element';
+import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
+import './styles.scss';
+
+export default function StripeAccountPanel({
```

Comment:
I'm thinking we should move this whole resources folder to be inside `Gateways\Stripe\StripePaymentElementGateway`.  That is - if these are specific to the payment element gateway - if so i'd rather keep them together.

Reply:
Sure